metadata-ws/pom.xml

```diff
@@ -14,6 +14,10 @@
         <version>1.0-SNAPSHOT</version>
     </parent>
 
+    <properties>
+        <cucumber.version>2.4.0</cucumber.version>
```

Comment:
why not use the latest version?

Reply:
Using 4.2.6 version gives build errors. Is there any compatibility issue?

Reply:
I think so.

Reply:
There are compatibility issues with higher versions for Cucumber and the version of Spring Boot that is currently used. We will address this once we come around to upgrade the Spring Boot version. So, imho, this conversion can be resolved.